app/js/components/manageApps/Help.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Link } from 'react-router';
+import checkforupdates from '../../../img/checkforupdates.png';
+import updateslist from '../../../img/updateslist.png';
+import view from '../../../img/view.png';
+import start from '../../../img/start.png';
+import stop from '../../../img/stop.png';
+
```

Comment:
I think it might be better to place the images in an assets folder that will contain all the static files needed by the app, it makes it easier to add more resources to the app like fonts etc.